diff --git a/classification_models/models/resnet.py b/classification_models/models/resnet.py
index d1f0861..9a401bc 100644
--- a/classification_models/models/resnet.py
+++ b/classification_models/models/resnet.py
@@ -225,7 +225,7 @@ def ResNet(model_params, input_shape=None, input_tensor=None, include_top=True,
     no_scale_bn_params = get_bn_params(scale=False)
     bn_params = get_bn_params()
     conv_params = get_conv_params()
-    init_filters = 64
+    init_filters = 16
 
     # resnet bottom
     x = layers.BatchNormalization(name='bn_data', **no_scale_bn_params)(img_input)
